src/validator/MaintainValidator.cpp

```diff
@@ -16,6 +17,51 @@
 
 namespace nebula {
 namespace graph {
+
+Status SchemaValidator::validateColumns(const std::vector<ColumnSpecification*> &columnSpecs,
+                                        meta::cpp2::Schema &schema) {
+    auto status = Status::OK();
+    std::unordered_set<std::string> nameSet;
+    for (auto& spec : columnSpecs) {
+        if (nameSet.find(*spec->name()) != nameSet.end()) {
+            return Status::SemanticError("Duplicate column name `%s'", spec->name()->c_str());
+        }
+        nameSet.emplace(*spec->name());
+        meta::cpp2::ColumnDef column;
+        auto type = spec->type();
+        column.set_name(*spec->name());
+        column.set_type(type);
+        column.set_nullable(spec->isNull());
+        if (meta::cpp2::PropertyType::FIXED_STRING == type) {
+            column.set_type_length(spec->typeLen());
+        }
+
+        if (spec->hasDefaultValue()) {
+            if (!evaluableExpr(spec->getDefaultValue())) {
+                return Status::SemanticError("Wrong default value experssion `%s'",
```

Comment:
More clear error message , such as `Constant expression required`

Reply:
`Constant expression` is the internal name, we just tell the user where it error